x-pack/plugins/security_solution/cypress/integration/header/search_bar.spec.ts

```diff
@@ -30,4 +30,14 @@ describe('SearchBar', () => {
       `${getHostIpFilter().key}: ${getHostIpFilter().value}`
     );
   });
+
+  it('popluates the filters correctly from the URL', () => {
+    const testFilter = getHostIpFilter();
+    const filterParam = `sourcerer=(default:(id:security-solution-default,selectedPatterns:!('logs-*')))&filters=!(('$state':(store:appState),meta:(alias:!n,disabled:!f,key:${testFilter.key},negate:!f,params:(query:'${testFilter.value}'),type:phrase),query:(match_phrase:(${testFilter.key}:'${testFilter.value}'))))`;
```

Comment:
I was wondering if there is a better way to generate this string rather than like this?